From aa0a63209af6813d87255ec3ab339f2dbbf27d6d Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 6 Mar 2017 13:38:46 -0800
Subject: [PATCH] Support Atomic ops on clang

clang pretends to be gcc 4.2 which is a lie
it actually supports a lot more features then
gcc 4.2, here it depends on gcc 4.7 to enable
the atomics and fails for clang

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Upstream-Status: Pending

---
 src/base/atomicops.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/base/atomicops.h b/src/base/atomicops.h
index dac95be..390733c 100644
--- a/src/base/atomicops.h
+++ b/src/base/atomicops.h
@@ -124,7 +124,7 @@
 #include "base/atomicops-internals-linuxppc.h"
 #elif defined(__GNUC__) && defined(__mips__)
 #include "base/atomicops-internals-mips.h"
-#elif defined(__GNUC__) && GCC_VERSION >= 40700
+#elif defined(__GNUC__) && GCC_VERSION >= 40700 || defined(__clang__)
 #include "base/atomicops-internals-gcc.h"
 #elif defined(__clang__) && CLANG_VERSION >= 30400
 #include "base/atomicops-internals-gcc.h"
